### PR TITLE
Fixing Google Codejam link, removing dead kickstart page

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @gaerae 

just fixing a link here and removing another one which doesn't exist anymore..

Cheers!

### CODE REVIEW

File: readme-en.md [hunk 1]
```diff
@@ -132,8 +132,7 @@ Language: **[😎 English](readme-en.md)** • [한국어 (Korean)](readme-ko.md
 - [Study data for algorithm](http://d2.naver.com/news/4850628) - Korean.
 
 ## Competitions
-- [Google Code Jam](https://code.google.com/codejam) - Google’s largest coding competition, calls on programmers around the world to put their skills to the test by solving multiple rounds of algorithmic puzzles.
-- [Google Code Jam Kickstart](https://code.google.com/codejam/kickstart) - University students looking to develop their coding skills and potentially pursue a Google career.
```
Comment (safv12) [inline readme-en.md:136]: Should we use this one to replace kickstart?

https://codingcompetitions.withgoogle.com/kickstart

Comment (spekulatius) [reply readme-en.md:136]: Oh, thank you @safv12. I've looked around and didn't see this. Thanks for pointing me to this! I've fixed it!

**Comment by spekulatius** [comment]
Readded the link for kickstart thanks to @safv12 now...

**Review by safv12 — APPROVED** [review]

**Comment by gaerae** [comment]
Sounds good, thanks @spekulatius @safv12 👍

**Comment by spekulatius** [comment]
Do you think it's ready to be merged @gaerae ?

**Comment by gaerae** [comment]
@spekulatius Please check it out.

Rename
- Google Code Jam Kickstart -> Google Kick Start
- Google #Hash Code -> Google Hash Code

Change link
https://hashcode.withgoogle.com -> https://codingcompetitions.withgoogle.com/hashcode

**Comment by spekulatius** [comment]
Done @gaerae

**Pull request merged**